lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -19,8 +19,9 @@ class AwsCompileSQSEvents {
           type: 'object',
           properties: {
             arn: { $ref: '#/definitions/awsArn' },
-            batchSize: { type: 'integer', minimum: 1, maximum: 10 },
+            batchSize: { type: 'integer', minimum: 1, maximum: 10000 },
             enabled: { type: 'boolean' },
+            maximumBatchingWindow: { type: 'integer', minimum: 0 },
```

Comment:
Let's also configure `maximum` (If I see correctly it's `300` per docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html#cfn-lambda-eventsourcemapping-maximumbatchingwindowinseconds)

Reply:
Good catch! I've added the maximum value check as well as the unit test